### PR TITLE
cards variation 2 with CTA

### LINKED ISSUE #6: [Homepage] Cards Block - Variation 2
- [ ] Default white background
- [ ] The card itself is clickable and has a CTA button too

Screenshot 2024-07-12 at 16 32 59

Mobile view:
Screenshot 2024-07-12 at 16 33 43

### PR DESCRIPTION
**Pull request by mind-adobe**
Fix #6 

Test URLs:
- Before: https://main--piramal--aemsites.hlx.page
- After: https://6-homepage-cards-block-variation-2--piramal--aemsites.hlx.page

### CODE REVIEW

**Comment by asthabh23** [comment]
@mind-adobe , is this PR ready for review? or its still WIP?
please do tag for review once its ready

File: blocks/cards/cards.css [hunk 1]
```diff
@@ -72,10 +77,84 @@
     align-self: center;
   }
 
+  .block.cards.image-cards {
+    flex-wrap: nowrap !important;
```
Comment (asthabh23) [inline blocks/cards/cards.css:81]: we preferably don't use `!important` .

File: blocks/cards/cards.css [hunk 2]
```diff
@@ -72,10 +77,84 @@
...
+  opacity: 1;
+  height: unset;
+  width: unset;
+  position: initial;
+  font-size: 0.75rem;
+  z-index: 1;
+}
+
+.block.cards.image-cards .url a {
+  background-color: #f26841;
```
Comment (asthabh23) [inline blocks/cards/cards.css:145]: This is giving accessibility issue:
Please try to use code: `A42E0E` for background color to fix the color contrast issue

**Review by asthabh23 — CHANGES_REQUESTED** [review]
Animation is missing in cards - `zoom-in` feature as on original site

File: blocks/cards/cards.js [hunk 3]
```diff
@@ -5,11 +5,24 @@
 export default function decorate(block) {
   const children = [...block.children];
   children.forEach((card) => {
-    const [img, title, url] = card.querySelectorAll(':scope > div > *');
-
+    let img;
+    let title;
+    let url;
+    if (block.classList.contains('image-cards')) {
+      console.log(img, title, url);
```
Comment (asthabh23) [inline blocks/cards/cards.js:12]: console statement not required.

File: blocks/cards/cards.css [hunk 4]
```diff
@@ -72,10 +84,84 @@
...
+  height: 100%;
+  object-fit: cover;
+  opacity: 0.9;
+  z-index: -1;
+}
+
+.block.cards.image-cards .card:hover .card-img img {
+  width: 100%;
+  height: 100%;
+  transform: scale(1.2);
```
Comment (asthabh23) [inline blocks/cards/cards.css:127]: the `zoom-in ``zoom-out` effect needs to be smooth. maybe an `ease-in-out` will help
please check on the original site once.

**Review by asthabh23 — CHANGES_REQUESTED** [review]

File: blocks/cards/cards.css (hunk 4 above)
Comment (mind-adobe) [reply blocks/cards/cards.css]: It's just a linear transition, I added it now.

**Review by asthabh23 — APPROVED** [review]

**Pull request merged**